apps/common-app/src/examples/RuntimeTests/tests/layoutAnimations/keyframe/basic.test.tsx

```diff
@@ -0,0 +1,193 @@
+import { StyleSheet, View } from 'react-native';
+import Animated, { Easing, Keyframe } from 'react-native-reanimated';
+import React from 'react';
+import {
+  describe,
+  test,
+  render,
+  expect,
+  clearRenderOutput,
+  mockAnimationTimer,
+  mockWindowDimensions,
+  recordAnimationUpdates,
+  unmockAnimationTimer,
+  unmockWindowDimensions,
+  waitForAnimationUpdates,
+} from '../../../ReJest/RuntimeTestsApi';
+import type { ReanimatedKeyframe } from 'react-native-reanimated/lib/typescript/layoutReanimation/animationBuilder/Keyframe';
+import { Snapshots } from './basic.snapshot.test';
+
+const AnimatedComponent = ({ enteringAnimation }: { enteringAnimation: ReanimatedKeyframe }) => {
+  return (
+    <View style={styles.container}>
+      <Animated.View style={styles.box} entering={enteringAnimation} />
+    </View>
+  );
+};
+
+describe('entering with custom animation (withDelay + withTiming color changes) test', () => {
+  const twirlingAnimation = new Keyframe({
+    0: { opacity: 0, transform: [{ translateY: 50 }, { rotate: '820deg' }, { scale: 0 }] },
+    50: {
+      opacity: 0.5,
+      transform: [{ translateY: 25 }, { rotate: '-180deg' }, { scale: 0.5 }],
+      easing: Easing.out(Easing.quad),
+    },
+    100: { opacity: 1, transform: [{ translateY: 0 }, { rotate: '0deg' }, { scale: 1 }] },
+  }).duration(1000);
+
+  const rainbowPulsarAnimation = new Keyframe({
+    0: { transform: [{ scale: 2 }], backgroundColor: 'red' },
+    15: { transform: [{ scale: 1 }], easing: Easing.poly(1) },
+    30: { transform: [{ scale: 2 }], easing: Easing.poly(2), backgroundColor: 'gold' },
+    45: { transform: [{ scale: 1 }], easing: Easing.poly(3) },
+    60: { transform: [{ scale: 2 }], backgroundColor: 'blue', easing: Easing.poly(10) },
+    75: { transform: [{ scale: 1 }], easing: Easing.poly(4) },
+    85: { transform: [{ scale: 2 }], backgroundColor: 'purple', easing: Easing.poly(5) },
+    100: { transform: [{ scale: 1 }], backgroundColor: 'red', easing: Easing.poly(6) },
+  }).duration(1000);
```

Comment:
Where did you get all these animations from?

Reply:
I've defined these animations trying to create something looking similar to real life examples, and including the following aspects:

- animation of array transforms, order of them is important
- animation of colors, defined with verious standards (color name, rgb, hsl)
- border radius animation
- easing

Reply:
I asked because if those animations were copied somewhere from the web, it would be nice to leave a reference here 😅 but if they're yours, then it's all good!